rbac/common/manager/rbac_manager.py

```diff
@@ -13,11 +13,90 @@
 # limitations under the License.
 # -----------------------------------------------------------------------------
 
+import logging
+import inspect
+
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from rbac.common.protobuf.rbac_payload_pb2 import RBACPayload
 from rbac.common.user.user_manager import UserManager
 from rbac.common.role.role_manager import RoleManager
+from rbac.common.task.task_manager import TaskManager
+
+LOGGER = logging.getLogger(__name__)
+
+
+def walk_filter(value):
+    return str(type(value)).startswith("<class 'rbac.common.")
+
+
+def hasgetattr(value, attrib):
+    if hasattr(value, attrib):
+        return getattr(value, attrib)
+    return None
+
+
+def message_type_name(message_type):
+    items = RBACPayload.MessageType.DESCRIPTOR.values_by_name.items()
+    # return items[message_type][0] # breaks on OSX and/or Python 3.7
+    for (key, value) in items:
+        if value == message_type:
+            return key
+    return None
 
 
 class RBACManager:
     def __init__(self):
         self.user = UserManager()
         self.role = RoleManager()
+        self.task = TaskManager()
+        self.handlers = {}
+        self.walk_handlers(self, "rbac")
+
+    def walk_handlers(self, node, path):
```

Comment:
Should this be a private function (ie __walk_handlers)?

Reply:
changed to ```_walk_handlers```